packages/celotool/src/e2e-tests/utils.ts

```diff
@@ -171,31 +174,17 @@ export function getHooks(gethConfig: GethRunConfig) {
 }
 
 export function getContext(gethConfig: GethRunConfig, verbose: boolean = verboseOutput) {
-  const mnemonic =
-    'jazz ripple brown cloth door bridge pen danger deer thumb cable prepare negative library vast'
   const validatorInstances = gethConfig.instances.filter((x: any) => x.validating)
```

Comment:
Nit: The following would clarify that only the count is needed here.
```suggestion
  const numValidators = gethConfig.instances.filter((x: any) => x.validating).length
```